CTP/README.md

```diff
@@ -36,18 +33,38 @@ This ``Quick Start`` is only for user for reference about how to use ``CTP`` to
 	URL:  http://127.0.0.1:8888    
     ```
 	
-* Please open the URL with your browser.
+* Please open the URL with your browser
   
 ## How To Execute
-- **SQL** 
+- **SQL**
  - Prepare
-	* Checkout test cases from our GitHub projects or make your own test cases.
-	* Install CUBRID and make sure your environment variable of ``CUBRID`` is correctly set.
-	* Check configuration files, for **SQL/Medium** test, you can modify parameters of ``conf/sql.conf`` or ``conf/medium.conf``. 
-	* Set ``scenario`` of ``[sql]`` section to the test cases directory.
-	* Set ``data_file`` of ``[sql]`` section to the directory path of initial data files for **Medium** test.
-	* ``test_mode=yes`` and ``java_stored_procedure=yes`` parameters must be set for **SQL/Medium** test.
-	* Please see ``conf/sql.conf`` or  ``conf/medium.conf`` for details about other parameters.
+ 	* Install CUBRID and make sure your environment variable of ``CUBRID`` is set correctly
+ 	* Check out scenarios from CUBRID GitHub projects or prepare your own test cases for testing
+ 	* Check out CTP and update the value of ``scenario`` parameter within ``CTP/conf/sql.conf`` to point to the path of your scenarios. For the current existing ``SQL`` test, you need to make sure the parameters `` java_stored_procedure=yes``, ``test_mode=yes`` and ``ha_mode=yes`` are configured
+ 	* **Example** ``sql.conf`` for scenario, data file and some important parameters changes
+ 	* 
+	  ```
+	  # SQL section - a section for CTP tool configuration when executing sql/medium testing
+	  [sql]
+
+	  # The location of your testing scenario
+	  scenario=${HOME}/cubrid-testcases/sql
+
+	  # SQL cubrid.conf section - a section for cubrid.conf configuration
+	  [sql/cubrid.conf]
+
+	  # To decide if the Java store procedure will be used when testing
+	  java_stored_procedure=yes
+
+	  # Allow scenario to change database system parameter
+	  test_mode=yes
+
+	  # In order to simulate the scenario customer uses
+	  ha_mode=yes
+	  ```
+ 	* For ``Medium`` test, ``data_file`` must be set to the path of the initial data file, regarding  the initial data file, please download it from [mdb.tar.gz](https://github.com/CUBRID/cubrid-testcases/tree/master/medium/files/mdb.tar.gz "mdb.tar.gz")
+
+	More parameters setting and parameters explanation within ``sql.conf``, please refer to [CTP/conf/sql.conf](conf/sql.conf)

```

Comment:
->
More parameters setting and parameters explanation within ``sql.conf`` and ``medium.conf``, please refer to [CTP/conf/sql.conf](conf/sql.conf) for SQL and [CTP/conf/medium.conf](conf/medium.conf) for Medium